rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -235,3 +249,161 @@ def test_migration_4_no_own_endpoint(rotkehlchen_api_server):
     with open(dir_path / 'data' / 'nodes.json', 'r') as f:
         nodes = json.loads(f.read())
         assert len(nodes) == len(web3_nodes)
+
+
+@pytest.mark.parametrize('data_migration_version', [None])
+@pytest.mark.parametrize('perform_migrations_at_unlock', [False])
+@pytest.mark.parametrize('perform_upgrades_at_unlock', [True])
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+def test_migration_5(rotkehlchen_api_server):
+    """
+    Test that the migration 5 works properly. What this migration does is:
+    - Change the combined_trades_view so a valid string is returned in the link field instead
+    of a blob.
+    """
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    database = rotki.data.db
+    migration_patch = patch(
+        'rotkehlchen.data_migrations.manager.MIGRATION_LIST',
+        new=MIGRATION_LIST[4:],
+    )
+
+    # Be sure to use the old view and insert one trade
+    swaps = [AMMSwap(
```

Comment:
Can you also add a normal trade and test both? Shoudl not change anything but the view you are touching should be for both.